config/config.default.php

```diff
@@ -47,7 +47,7 @@
     'display_news'            => 6,
 
     // Users are able to sign up
-    'registration_enabled'    => true,
+    'registration_enabled'    => env('REGISTRATION_ENABLED', 1),
```

Comment:
This should be a boolean and only a 0 when disabled with an environment variable